client-react/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
+    "@material-ui/icons": "^4.11.3",
```

Comment:
This version of icons is outdated and you need to use V5
@mui/icons-material
https://mui.com/material-ui/getting-started/installation/#svg-icons

Reply:
Added new package